docs-site/content/guide/install-typesense.md

```diff
@@ -134,9 +134,9 @@ ignore this error message , executing `apt list --installed | grep typesense` wo
 </Tabs>
 
 - The default API key is `xyz` and the default port is `8108`
-- The config file is at `/usr/local/etc/typesense/typesense.ini`
-- Logs are under `/usr/local/var/log/typesense/`
-- Data dir is under `/usr/local/var/lib/typesense/`
+- The config file is at `/opt/homebrew/etc/typesense/typesense.ini`
```

Comment:
@Vashiru This is actually the location that homebrew uses just on ARM-based M1 machines. It still uses the old directories for non-ARM machines. 

So instead of changing it, I think we should call out both - one section for Intel Macs and another for Apple Silicon Macs.